blocks/columns/columns.css

```diff
@@ -53,6 +53,13 @@
   margin: 0 auto 1rem;
 }
 
+.columns.heading-divider h2,
```

Comment:
I have merged these changes through [the other PR](https://github.com/hlxsites/sunstar/pull/333), please remove form here